docs/monitoring/05-03-prometheus.md

```diff
@@ -15,5 +15,6 @@ This table lists the configurable parameters, their descriptions, and default va
 
 | Parameter | Description | Default value |
 |-----------|-------------|---------------|
-| **retention** | Specifies a period for which Prometheus stores the metrics in-memory. This retention time applies to in-memory storage only. Prometheus stores the recent data in-memory for the specified amount of time to avoid reading the entire data from disk.| `2h` |
-| **storageSpec.volumeClaimTemplate.spec.resources.requests.storage** | Specifies the size of a Persistent Volume Claim (PVC). | `4Gi` |
+| **prometheusSpec.retention** | Specifies a period for which Prometheus stores the metrics.| `1d` |
+| **prometheusSpec.retentionSize** | Maximum number of bytes that storage blocks can use. The oldest data will be removed first.| `2GB` |
```

Comment:
```suggestion
| **prometheusSpec.retentionSize** | Specifies the maximum number of bytes that storage blocks can use. The oldest data will be removed first.| `2GB` |
```

Reply:
For consistency sake, start either with a verb or with a noun in the context of the whole document.